src/components/emptyStatePages/NoOrgApplicationLandingPage.tsx

```diff
@@ -16,14 +21,39 @@ const EMPTY_STATE_CONTENT_STYLES = {
   listContainerVerticalMargin: '48px',
 };
 
-type Prop = {
-  redirectTo?: string;
-};
-
-export default function NoOrgApplicationLandingPage({ redirectTo }: Prop): JSX.Element {
+export default function NoOrgApplicationLandingPage(): JSX.Element {
   const { isMobile } = useDeviceInfo();
   const theme = useTheme();
   const [isOrgModalOpen, setIsOrgModalOpen] = useState<boolean>(false);
+  const { goToApplication, goToApplicationList } = useNavigateTo();
+  const { toastError } = useSnackbar();
+
+  const onOrgCreated = useCallback(
+    async (organization: Organization) => {
+      // Create project with the same name
+      const projectResult = await ProjectsService.createProject({
+        name: organization.name,
+        organizationId: organization.id,
+      });
+
+      if (!(projectResult.data && projectResult.requestSucceeded)) {
+        toastError(strings.ERROR_CREATE_PROJECT);
+        goToApplicationList();
+        return;
+      }
+
+      // Create application with new project
+      const applicationResult = await ApplicationService.createApplication(projectResult.data.id);
+      if (!(applicationResult.data && applicationResult.requestSucceeded)) {
+        toastError(strings.ERROR_CREATE_APPLICATION);
+        goToApplicationList();
+        return;
+      }
```

Comment:
This seems like another set of requests that should be handled in a single service.

Reply:
Added compound requests and redux in this PR!
https://github.com/terraware/terraware-web/pull/2913

Will rebase and apply new calls in this PR.